Podcast/CardTableViewCellContextView.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  CardTableViewCellContextView.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 9/18/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class CardTableViewCellContextView: UIView {
+    
+    static var contextViewHeight: CGFloat = 52
+    
+    var lineSeperatorHeight: CGFloat = 1
+    var contextMarginX: CGFloat = 17
+    var contextLabelRightX: CGFloat = 20
+    var contextImagesSize: CGFloat = 28
+    var feedControlButtonRightX: CGFloat = 20
+    var feedControlButtonHieght: CGFloat = 7.5
+    var feedControlButtonWidth: CGFloat = 13
+    var contextViewHeight: CGFloat = CardTableViewCellContextView.contextViewHeight
+    var marginSpacing: CGFloat = 10
+    ///
+    /// Mark: Variables
+    ///
+    var topLineSeperator: UIView!
+    var contextLabel: UILabel!
+    var contextImages: UIStackView!
+    var feedControlButton: FeedControlButton!
+   
+    ///
+    ///Mark: Init
+    ///
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = .podcastWhite
+        
+        contextLabel = UILabel(frame: CGRect.zero)
+        contextLabel.textAlignment = .left
+        contextLabel.lineBreakMode = .byWordWrapping
+        contextLabel.font = UIFont.systemFont(ofSize: 12.0)
+        contextLabel.numberOfLines = 2
+        addSubview(contextLabel)
+
+        feedControlButton = FeedControlButton(frame: .zero)
+        addSubview(feedControlButton)
+        
+        topLineSeperator = UIView(frame: CGRect.zero)
+        topLineSeperator.backgroundColor = .podcastWhiteDark
+        addSubview(topLineSeperator)
+        
+        contextImages = UIStackView()
+        contextImages.spacing = -1 * contextImagesSize
```

Comment:
Does this work? I would think that the spacing needs to be a fraction of the size of the images